benchmarks/time.rs

```diff
@@ -207,5 +209,52 @@ setup_benchmark! {
         ben.iter(|| time!(12:00) > time!(11:00));
         ben.iter(|| Time::MIDNIGHT == time!(0:00:00.000_000_001));
     }
+
+    fn sort(ben: &mut Bencher<'_>) {
+        ben.iter_batched_ref(
+            || {
+                let mut res = vec![];
+                let mut last = Time::MIDNIGHT;
+                for _ in 0..4096 {
```

Comment:
With 4096 values, this only accounts for 68 minutes. Shouldn't the values be more spread out over the full range?

Reply:
My logic was that it is enough to have code using all attributes of the struct, but I can make it larger if you prefer.

Reply:
My thinking is that the CPU could take advantage of branch prediction to largely eliminate the check of the hour, assuming a different implementation. Benchmarks should be implementation-agnostic, after all.

Reply:
Ok I will make it larger then!

Reply:
And I'll note that it doesn't necessarily mean more iterations — having a larger value added each time would work as well.

Reply:
I've taken a slightly different route, because I felt like having a millisecond increment that is the closest prime to 1000 makes sense to go through every second and minute possible. I'm now incrementing hours independently by 5 (prime with 24) modulo 24. Hope that's fine.

Reply:
Yeah, that works as well. I was just seeking something that would result in the values being more spread out, and that does the job.